### DISCUSSION TITLE
AutoIncrement with start value?

### DISCUSSION THREAD

**Discussion by Imadlekal** [collerek/ormar#790] [Q&A]
So, I'm using ORMAR with my postgres database, I have my ID column as the primary key, where autoincrement is enabled.

The values being inserted however are starting from 1. Is there any way to make them start from a different value, say 10000?

**Comment by sbathe** [comment]
Have you tried specifying "min" in the definition ?
Something like:
`id: int = ormar.Integer(primary_key=True, autoincrement=True, min=10000)`
